mindmeld/components/entity_recognizer.py

```diff
@@ -71,6 +71,23 @@ def _get_model_config(self, **kwargs):  # pylint: disable=arguments-differ
         )
         return super()._get_model_config(loaded_config, **kwargs)
 
+    def get_entity_types(self, queries=None, label_set=None, **kwargs):
+
+        if not label_set:
+            label_set = self._get_model_config(**kwargs).train_label_set
+            label_set = label_set if label_set else DEFAULT_TRAIN_SET_REGEX
+
+        # Load labeled data
+        queries, labels = self._get_queries_and_labels(queries, label_set=label_set)
```

Comment:
This call will probably change once #301  changes are merged?

Reply:
Will look into it and see if needed to be changed.